package com.example.InstagramClone.Exception;

import java.time.LocalDateTime;

public class ErrorDetails {

    private String message;
    private String details;
    private LocalDateTime timeStamp;


    public ErrorDetails() {

    }


    public ErrorDetails(String message, String details, LocalDateTime timeStamp) {
        this.message = message;
        this.details = details;
        this.timeStamp = timeStamp;
    }


    
}
